drivers/kscan/Kconfig.sdl

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Jabil Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config KSCAN_SDL
+	bool "SDL event filter for touch panel emulation"
+	depends on SDL_DISPLAY
+	default y if SDL_DISPLAY
```

Comment:
I would not enable this by default even if `SDL_DISPLAY` is enabled

Reply:
got it. Done